pages/setup/windows/vmware-on-windows-linux/README.md

```diff
@@ -1,7 +1,7 @@
 ---
 navigation_title: "VMware on Windows & Linux"
 position: 4
-changed: "2020-04-30"
+changed: "2021-04-22"
```

Comment:
Dito

Reply:
Fixed in https://github.com/Access4all/adg/pull/326/commits/00d8698c03d5f5738b07366d9efc335694e446bc